AgentDiscoveries-Frontend/app/src/components/app.jsx

```diff
@@ -19,8 +19,11 @@ import RegionForm from './admin/region-form';
 import UserForm from './admin/user-form';
 import Error from './error';
 
+import {apiGet} from './utilities/request-helper.js';
+
 export default class App extends React.Component {
     render() {
+        apiGet('/checktoken');
```

Comment:
For the other group, this seemed to work but also display a console error - can you check whether that's the case here too?